homeassistant/components/evohome/__init__.py

```diff
@@ -273,10 +273,10 @@ def update(self, *args, **kwargs) -> None:
         else:
             self.timers["statusUpdated"] = utcnow()
 
-        _LOGGER.debug("Status = %s", status)
+            _LOGGER.debug("Status = %s", status)
 
-        # inform the evohome devices that state data has been updated
-        async_dispatcher_send(self.hass, DOMAIN, {"signal": "refresh"})
+            # inform the evohome devices that state data has been updated
+            async_dispatcher_send(self.hass, DOMAIN, {"signal": "refresh"})
```

Comment:
Good work! 

I think this is better, OK with you?

Reply:
Yes, that actually makes more sense